### PR TITLE
Fixed markup

### PR DESCRIPTION
**Pull request by zpawn**
Story https://app.shortcut.com/deskpro/story/151208/copper-improvements

### CODE REVIEW

File: testing/mockClient.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+const mockClient = {
```
Comment (AshleyDawson) [inline testing/mockClient.ts:1]: Think we should extract this back into the app SDK

**Review by AshleyDawson — APPROVED** [review]

File: src/App.tsx [hunk 2]
```diff
@@ -1,54 +1,43 @@
...
-import { Redirect } from "./components/Redirect/Redirect";
-import { query } from "./utils/query";
-import { View } from "./pages/View/View";
-import { VerifySettings } from "./pages/Admin/VerifySettings";
+  useRegisterElements(({ registerElement }) => {
+    registerElement("refresh", { type: "refresh_button" });
+  });
+
+  useDeskproAppEvents({
+    async onElementEvent(id) {
```
[the remark this answers is not in the document]
Comment (zpawn) [reply src/App.tsx:22]: Thanks, I'll remove `async` in the next PR

File: src/App.tsx [hunk 3]
```diff
@@ -1,54 +1,43 @@
...
-      </QueryClientProvider>
-    </HashRouter>
+    <>
+      <Routes>
+        <Route path="/admin/verify_settings" element={<VerifySettingsPage/>}/>
+        <Route path="/home" element={<HomePage/>}/>
+        <Route path="/opportunity/:id" element={<OpportunityPage/>}/>
+        <Route index element={<LoadingPage/>}/>
+      </Routes>
+      {!isAdmin && (<><br/><br/><br/></>)}
```
[the remark this answers is not in the document]
Comment (zpawn) [reply src/App.tsx:38]: I agree, it will be prettier to use CSS

File: testing/mockClient.ts (hunk 1 above)
Comment (zpawn) [reply testing/mockClient.ts:1]: done https://github.com/DeskproApps/app-testing-utils

**Pull request merged**